src/AbstractFunctionReferenceFixer.php

```diff
@@ -67,7 +67,7 @@ protected function find($functionNameToSearch, Tokens $tokens, $start = 0, $end
         $matches = $tokens->findSequence($candidateSequence, $start, $end, false);
         if (null === $matches) {
             // not found, simply return without further attempts
-            return;
+            return null;
```

Comment:
> Fatal error: A void function must not return a value

anyway, great to see you being active. would be great on focus on sth meaningful instead, eg https://github.com/FriendsOfPHP/PHP-CS-Fixer/pulls/GrahamCampbell